src/darwin/CHIPTool/CHIPTool/View Controllers/QRCode/QRCodeViewController.m

```diff
@@ -104,20 +116,24 @@ - (void)setupUI
 
     // Setup nav bar button
     [self changeNavBarButtonToCamera];
+    
+    // Initialize all Labels
+    [self initializeAllLabels];
 
     // Title
-    UILabel * titleLabel = [CHIPUIViewUtils addTitle:@"QR Code Parser" toView:self.view];
+    _titleLabel = [CHIPUIViewUtils addTitle:@"QR Code Parser" toView:self.view];
 
     // stack view
     UIStackView * stackView = [UIStackView new];
     stackView.axis = UILayoutConstraintAxisVertical;
     stackView.distribution = UIStackViewDistributionFill;
     stackView.alignment = UIStackViewAlignmentLeading;
-    stackView.spacing = 15;
+    stackView.spacing = 10;
```

Comment:
15 -> 10 because adding a new row messed up the UI and this is a workaround to show all infos. ScrollView for some reason is not working.